sdk/src/feathers/controls/TextInput.ls

```diff
@@ -133,6 +133,12 @@ package feathers.controls
          * @private
          */
         protected var _text:String = "";
+        
+        /**
+         * @private
+         * Par of a Workaround fix that specially prevents editing when not in focus
```

Comment:
I fixed a bunch of text input / focus bugs most likely after this was integrated, so I would say that it would be good to temporarily revert this piece of code, merge in master (if not already) and check if there are still any issues with input.